test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/MSBuildAPIUtilityTests.cs

```diff
@@ -431,5 +433,142 @@ public void UpdateVersionOverrideInPropsFileWhenItExists_Success()
             Assert.Contains(@$"<PackageVersion Include=""X"" Version=""1.0.0"" />", updatedPropsFile);
             Assert.DoesNotContain(@$"<PackageVersion Include=""X"" VersionOverride=""3.0.0"" />", updatedPropsFile);
         }
+
+        [Fact]
+        public void GetListOfProjectsFromPathArgument_WithProjectFile_ReturnsCorrectPaths()
+        {
+            // Arrange
+            var pathContext = new SimpleTestPathContext();
+
+            var net8 = NuGetFramework.Parse("net8.0");
+
+            var projectA = SimpleTestProjectContext.CreateNETCore(
+                "a",
+                pathContext.SolutionRoot,
+                net8);
+
+            projectA.Save();
+
+            // Act
+            var projectList = MSBuildAPIUtility.GetListOfProjectsFromPathArgument(projectA.ProjectPath);
+
+            // Assert
+            Assert.Equal(projectList.Count(), 1);
+
+            Assert.Contains(projectA.ProjectPath, projectList);
+        }
+
+        [Fact]
+        public void GetListOfProjectsFromPathArgument_WithProjectDirectory_ReturnsCorrectPaths()
+        {
+            // Arrange
+            var pathContext = new SimpleTestPathContext();
+
+            var net8 = NuGetFramework.Parse("net8.0");
+
+            var projectA = SimpleTestProjectContext.CreateNETCore(
+                "a",
+                pathContext.SolutionRoot,
+                net8);
+
+            projectA.Save();
```

Comment:
We can remove extra blank lines to save vertical space, especially when each line of code has a blank line between it and the next line of code. This comment applies to all tests.